### ISSUE TITLE
Opening Playnite on full screen on the taskbar always showed stuck taskbar after opening it

### ISSUE THREAD

**Issue by BenjiBenji69** [JosefNemec/Playnite#3360]
### Bug Description

Its a very minor issue, and forgive me if its bothering but it is a minor issue to me at least, its just taskbar always getting stuck on opening up Playnite fullscreen mode, it could be usual Windows bugs and i notice it did happen on some other apps that has "fullscreen mode" app or whatnot but i suppose it can be fixable

### To Reproduce

Pin the Playnite app on the taskbar on full screen mode and then opening it

### Diagnostics ID

4e8f5565-d997-4744-86f7-7985cff7c744

### Screenshots

<img width="960" alt="image" src="https://user-images.githubusercontent.com/99275631/228487772-e98162b3-4f85-4dd8-a493-4c1326ad69c2.png">

**Comment by JosefNemec** [comment]
Nothing we can do about it. It's a Windows thing, alt-tabbing usually fixes it.

**Comment by BenjiBenji69** [comment]
yeah i'm aware of that, its weird though sometimes it doesn't sometimes it does, i notice it often happens with apps that opens up new tasks while it opens up the fullscreen app or some other tasks, it happens with any apps that does that the likes of Kodi when changing themes and even Retroarch when you click the taskbar, its a long running  Windows issues that even make Automatically Hide Taskbar option made the taskbar stuck instead. i hope Microsoft fix this

the only fix for this is just to pin the purple Fullscreen Playnite app instead. sorry bothering you and thanks for responding

**Comment by JosefNemec** [comment]
While in theory there could be a possibility to add some workaround, the main issue is detecting that the issue actually occurred. I don't think there is any programmatically way of detecting if task bar is wrongly rendering over your app. And using any image recognition based solutions is something I really do not like.

It's one of those Windows bugs that's been there since forever. Like for example task bar sometimes still showing tray icons of apps that are no longer running.

**Comment by BenjiBenji69** [comment]
i think just maybe some fix for it make it so that opening up fullscreen  on the taskbar does not open a separate tasks or windows, it does cause that when opening and pinning the default desktop icons where it opens also opens the fullscreen app also but not on only pinning the purple one

 i didnt even know theres separate exe or tasks for fullscreen mode for Playnite (PlayniteFullscreen.App.exe) before i pin it on the taskbar, i suppose because its a sandbox  for each modes, but i think Retroarch does not have that issues often because it does not separate the two sandboxes modes,i think its not too technical to make a work around at least in the app itself but its just suggestion and i think there should be FAQs or notes that tells people to just pin the fullscrreen app instead on the taskbar to avoid the issue sionce the fullscreen app does not pinned when you installed the app and its not obvious to some

sorry for the ramblings for such minor issue, it is still one of those Windows issues

**Comment by Loddio** [comment]
Any work around to fix this? Quite pointless to use the fullscreen mode with this

**Comment by Jef-RS** [comment]
To resolve this, go to **"properties"** in the Playnite application and set it to **"Maximized"** instead of **"Window mode"**,

**Comment by Jef-RS** [comment]
> > To resolve this, go to **"properties"** in the Playnite application and set it to **"Maximized"** instead of **"Window mode"**,
> 
> This does not help.  I resorted to running a script every time a game closes and anytime Playnite starts to wait 5 seconds and then re-focus the Playnite window, which puts it in front of the taskbar.

For me it worked. I apply it to the blue Playnite shortcut (which already has fullscreen mode), I don't know if that can make a difference

**Comment by carlosdesedas** [comment]
> > To resolve this, go to **"properties"** in the Playnite application and set it to **"Maximized"** instead of **"Window mode"**,
> 
> This does not help. I resorted to running a script every time a game closes and anytime Playnite starts to wait 5 seconds and then re-focus the Playnite window, which puts it in front of the taskbar.

can you please share the script?  I have the same problem!  thanks

**Comment by SudipGhosh822** [comment]
> Any work around to fix this? Quite pointless to use the fullscreen mode with the taskbar like this in my opinion

Yes you need a ahk script. I fixed it properly. If you want to know about tell me i will describe all of process.☺️

**Comment by eight-track** [comment]
> > Any work around to fix this? Quite pointless to use the fullscreen mode with the taskbar like this in my opinion
> 
> Yes you need a ahk script. I fixed it properly. If you want to know about tell me i will describe all of process.☺️

Can you let me know how you fixed this?

**Comment by daemonsloth** [comment]
As a workaround, I’ve used AntiMicroX to assign the Guide button to Alt+Tab on my Xbox controller. So whenever that happens, a single tap on the Guide button fixes it.

This also comes in handy for switching focus between the game window and Playnite when needed.
